frontend/src/components/common/ReleaseNotes/ReleaseNotes.tsx

```diff
@@ -21,7 +21,9 @@ export default function ReleaseNotes() {
             return;
           }
 
-          const octokit = new Octokit();
+          const octokit = new Octokit({
+            timeout: 5000,
+          });
```

Comment:
Was this the one place causing the delay?
Could we also get in trouble by fetching the actual release ( 2 lines below) if the network gets hyper slow then?

Reply:
So that is handled in, this object only every request made using this instance will have a timeout of 5 ms

Reply:
Okay.